### PR TITLE
fix: update login screen, texture guide, background colors, divider

### PR DESCRIPTION
**Pull request by paulschreiber**
## Description
* update login screen to match [new designs](https://www.figma.com/design/h4xdqfTq1c84YEn6AZcfHf/Splash-Screen-to-Log-In-Animation?node-id=2-56&t=31mUduoewqR4pY3P-0) (add logo, update text strings, left-aligned, green, uppercase button text (on Android))
* fix text input field background color https://github.com/techmatters/terraso-mobile-client/issues/688
* in texture guide, move videos above questions and reduce question font size https://github.com/techmatters/terraso-mobile-client/issues/1535#issuecomment-2189430152
* add divider to slope screen inputs https://github.com/techmatters/terraso-mobile-client/issues/1640

### CODE REVIEW

File: dev-client/src/screens/LoginScreen.tsx [hunk 1]
```diff
@@ -125,22 +128,26 @@ export const LoginScreen = () => {
...
+          <TerrasoLogo width="122px" height="39px" />
+        </Row>
       </Column>
     </Column>
   );
 };
 
 const styles = StyleSheet.create({
+  loginButton: {
+    backgroundColor: 'white',
```
Comment (paulschreiber) [inline dev-client/src/screens/LoginScreen.tsx:144]: This doesn't work with `background.default` or `primary.contrast`. Would like to understand why/fix.

File: dev-client/src/screens/SlopeScreen/SlopeScreen.tsx [hunk 2]
```diff
@@ -68,14 +69,15 @@ export const SlopeScreen = ({siteId}: {siteId: string}) => {
       <DataInputSummary
```
Comment (knipec) [inline dev-client/src/screens/SlopeScreen/SlopeScreen.tsx:69]: Issue: It appears the Figma design calls for a divider here too
```suggestion
      <Divider />
      <DataInputSummary
```

File: dev-client/src/screens/LoginScreen.tsx
Comment (knipec) [inline dev-client/src/screens/LoginScreen.tsx:1]: Issue: Looks a little bottom-heavy on my iOS emulator device -- is this expected? 
Wonder if we may want to treat all the content from the logo down to the buttons as a box, and use flex or center to position that between the top and the "from Terraso" content.
Screenshot 2024-06-27 at 11 26 28 AM

Comment (paulschreiber) [reply dev-client/src/screens/LoginScreen.tsx:1]: I reworked the login screen layout to better support multiple screen sizes:
image

**Review by knipec — APPROVED** [review]

**Comment by paulschreiber** [comment]
Here's an iPad version for completeness:
image

**Pull request merged**